crates/pink/pink-extension/src/lib.rs

```diff
@@ -224,27 +303,61 @@ pub fn force_stop_sidevm() {
     emit_event::<PinkEnvironment, _>(PinkEvent::StopSidevm)
 }
 
-/// Push a message to the associated sidevm instance.
+/// Pushes a message to the associated SideVM instance.
+///
+/// Note: There is no guarantee that the message will be received by the SideVM instance.
+/// The message may be dropped due to several reasons:
+///
+/// - The SideVM instance is not currently running.
+/// - The SideVM instance is running, but the message queue is full. This may occur when the SideVM
+///   instance is busy processing other messages.
+///
+///# Arguments
+///
+///* `message`: The message to be pushed to the SideVM instance.
 pub fn push_sidevm_message(message: Vec<u8>) {
     emit_event::<PinkEnvironment, _>(PinkEvent::SidevmMessage(message))
 }
 
-/// Set the log handler contract of current cluster
+/// Set the log handler contract of current cluster. (system only)
 pub fn set_log_handler(contract: AccountId) {
     emit_event::<PinkEnvironment, _>(PinkEvent::SetLogHandler(contract))
 }
 
-/// Set the weight of contract used to schedule queries and sidevm vruntime
+/// Set the weight of contract used to schedule queries and sidevm vruntime. (system only)
```

Comment:
Does `vruntime` mean something like `virtual runtime`? I see it used twice.

Reply:
Yes. Replaced with `virtual runtime`.